prometheus/promhttp/delegator.go

```diff
@@ -147,45 +147,45 @@ func init() {
 			*responseWriterDelegator
 			io.ReaderFrom
 			http.CloseNotifier
-		}{d, &readerFromDelegator{d}, &closeNotifierDelegator{d}}
+		}{d, &readerFromDelegator{d}, closeNotifierDelegator{d}}
```

Comment:
`&readerFromDelegator` should now also work as simply `readerFromDelegator`, shouldn't it? Same for the other occurrences below.